src_ts/etools-dropdown/SlAutocomplete.ts

```diff
@@ -279,11 +284,12 @@ export class SlAutocomplete extends LitElement {
   private previousSelMenuItemElem?: SlMenuItem;
 
   render() {
-    const hasHelpText = this.helpText ? true : false;
+    const hasHelpText = !!this.helpText;
     const hasClearIcon =
       this.clearable && this.multiple && !this.disabled && !this.readonly && this.selectedValueCommaList.length > 0;
     const isPlaceholderVisible = this.placeholder && this.selectedValueCommaList.length === 0;
     // this.filteredOptions should be called only once otherwise it breaks pagination.
+    this.totalOptionsToShow = this.shownOptionsLimit;
```

Comment:
This needs to be reverted cause otherwise virtualscroll is not working anymore. And instead maybe we should add a new property disableVirtualScroll and set this on the country dropdown

Reply:
Requested modifications, implemented.